subsys/caf/modules/leds.c

```diff
@@ -200,11 +200,14 @@ static int leds_init(void)
 static void leds_start(void)
 {
 	for (size_t i = 0; i < ARRAY_SIZE(leds); i++) {
-#ifdef CONFIG_PM_DEVICE
-		int err = pm_device_state_set(leds[i].dev,
-					      PM_DEVICE_STATE_ACTIVE);
+	/* Zephyr power management API is not implemented for GPIO LEDs.
```

Comment:
alignment is wrong
Also move comment under check.

To be honest I am not sure what to do with this one. Why we cannot check the driver response but have to explicitly know what it supports?

Reply:
Fixed alignment and moved comment under check

Reply:
It seems we have no choice but that is something that is strange. By compliance you mean checkpatch? Have you checked with @carlescufi ?

Reply:
> Also move comment under check.

I have not given the rationale and this may be useful to understand. Some times there is a need to run a tool like `unifdef`. If you place anything that belongs to section that is to be eliminated outside of this section, you get leftovers. This is not a problem for vim users but GUIs that hide preprocessor conditional content will also show leftovers.
Sorry if this is obvious.

Reply:
Added https://github.com/nrfconnect/sdk-nrf/pull/5782 to replicate the issue and as a followup.